lms/djangoapps/certificates/views.py

```diff
@@ -481,9 +468,19 @@ def render_html_view(request, user_id, course_id):
     configuration = CertificateHtmlViewConfiguration.get_config()
     context.update(configuration.get('default', {}))
 
+    # Translators:  This is the copyright line which appears at the bottom of the certificate page/screen
+    context['copyright_text'] = _('&copy; {year} {platform_name}. All rights reserved.').format(
+        year=datetime.now().year,
```

Comment:
This should use [`settings.COPYRIGHT_YEAR`](https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L51).